x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/optimizer/Optimizer.java

```diff
@@ -1097,12 +1097,12 @@ private boolean canPropagateFoldable(LogicalPlan p) {
         @Override
         protected Expression rule(Expression e) {
             if (e instanceof IsNotNull) {
-                if (((IsNotNull) e).field().nullable() == false) {
+                if (((IsNotNull) e).field().nullable() == Nullability.FALSE) {
```

Comment:
Can you static import this one?

Reply:
Cannot, mixes up with `Literal.TRUE` and `Literal.FALSE` which are already statically imported.